### DISCUSSION TITLE
怎么修改图例？

### DISCUSSION THREAD

**Discussion by TTsXDU** [wh201906/SerialTest#55] [Q&A] [answered]
请问我想在上位机运行后显示的图例中直接显示我想要编辑的文本，而不是默认的Graph1Graph2，该怎么修改代码呢？（是运行后直接显示，不是运行后点击图例进行修改）
![QQ截图20240305111521](https://github.com/wh201906/SerialTest/assets/121789556/e3b65cc7-f81e-4fba-8981-402e0b9802e6)

**Comment by wh201906** [comment, marked as answer]
程序会自动保存图例的。
你首次运行之后修改图例名称，就会生成对应的配置文件，下次运行的时候就会直接显示了。如果有打包的需求就连配置文件一起打包。

**Reply by TTsXDU** [reply]
按你说的改好了，感谢
